.github/workflows/build-test-macos.yml

```diff
@@ -0,0 +1,69 @@
+# This is a basic workflow to help you get started with Actions
+
+name: MacOS-CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, devel ]
+  pull_request:
+    branches: [ master, devel ] # Can dd back release/v3.0.0 once SBT is install in the build system
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  MacOS-Framework:
+    runs-on: macos-latest
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    - name: Install fprime tools
+      run: pip3 install fprime-tools fprime-gds
+    - name: F prime CI step
+      run: ./ci/tests/Framework.bash
+    # Archive the outputs
+    - name: 'Archive Logs'
+      uses: actions/upload-artifact@v2
+      if: always()
+      with:
+        name: ci-framework-logs
+        path: ci-logs.tar.gz
+        retention-days: 5
+
+  MacOS-Ref:
+    runs-on: macos-latest
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    - name: Install fprime tools
+      run: pip3 install fprime-tools fprime-gds
+    - name: F prime CI step
+      run: ./ci/tests/Ref.bash
+    # Archive the outputs
+    - name: 'Archive Logs'
+      uses: actions/upload-artifact@v2
+      if: always()
+      with:
+        name: ci-ref-logs
+        path: ci-logs.tar.gz
+        retention-days: 5
+
+  MacOS-Integration:
+    runs-on: macos-latest
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    - name: Install fprime tools
+      run: pip3 install fprime-tools fprime-gds
+    - name: Setup Dependencies
+      run: sudo apt-get install valgrind
```

Comment:
This isn't gonna work :)

Reply:
Yeah, I blew up on that one.  Should be fixed.